### PR TITLE
Added Google Code-In 2018 Link

### PR DESCRIPTION
**Pull request by niranjannitesh**
Fixes https://github.com/fossasia/fossasia.org/issues/345

Added Google Code-In 2018 Link in website.

screenshot from 2018-10-28 11-25-42

### CODE REVIEW

**Review by akshatnitd — COMMENTED** [review]
Please link the issue and fill in the PR description

**Review by Akshat-Jain — APPROVED** [review]
LGTM 👍

**Review by akshatnitd — COMMENTED** [review]
Provide a demo link and also change `Solving` to `Fixes` in the PR description so that the issue is auto closed on merge.

**Comment by niranjannitesh** [comment]
@akshatnitd I can't give demo link because I'm creating this PR from other branch. but I will post screenshot.

File: index.html [hunk 1]
```diff
@@ -83,7 +83,7 @@
 											<li><a target="_self" href="/adp">Advanced Developer Program</a></li>
 										</ul>
 									<li class="has-dropdown"><a href="https://labs.fossasia.org/#projects">Projects</a>
-               					     <ul class="nav-dropdown projects" style="min-height: 590px"> <!--whenever you add/remove an option, change the style="min-height:500px" by +- 25px-->
+               					     <ul class="nav-dropdown projects" style="min-height: 605px"> <!--whenever you add/remove an option, change the style="min-height:500px" by +- 25px-->
```
Comment (abishekvashok) [inline index.html]: Add 25px not 15px

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by niranjannitesh** [comment]
@abishekvashok Done!

**Review by abishekvashok — APPROVED** [review]

**Comment by abishekvashok** [comment]
Thank you

**Pull request merged**